
Epoch: 0
Traceback (most recent call last):
  File "/mnt/c/Users/tandf/OneDrive - 電気通信大学/Src/DVRL/train_MLP.py", line 150, in <module>
    main(args)
  File "/mnt/c/Users/tandf/OneDrive - 電気通信大学/Src/DVRL/train_MLP.py", line 120, in main
    dev_qwk = calc_qwk(y_test, y_pred_dev, test_prompt_id, 'score')
  File "/mnt/c/Users/tandf/OneDrive - 電気通信大学/Src/DVRL/utils/dvrl_utils.py", line 127, in calc_qwk
    return cohen_kappa_score(y_true, y_pred, weights='quadratic', labels=[i for i in range(minscore, maxscore+1)])
  File "/home/takumi/anaconda3/envs/torch/lib/python3.10/site-packages/sklearn/utils/_param_validation.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/home/takumi/anaconda3/envs/torch/lib/python3.10/site-packages/sklearn/metrics/_classification.py", line 680, in cohen_kappa_score
    confusion = confusion_matrix(y1, y2, labels=labels, sample_weight=sample_weight)
  File "/home/takumi/anaconda3/envs/torch/lib/python3.10/site-packages/sklearn/utils/_param_validation.py", line 184, in wrapper
    return func(*args, **kwargs)
  File "/home/takumi/anaconda3/envs/torch/lib/python3.10/site-packages/sklearn/metrics/_classification.py", line 326, in confusion_matrix
    y_type, y_true, y_pred = _check_targets(y_true, y_pred)
  File "/home/takumi/anaconda3/envs/torch/lib/python3.10/site-packages/sklearn/metrics/_classification.py", line 84, in _check_targets
    check_consistent_length(y_true, y_pred)
  File "/home/takumi/anaconda3/envs/torch/lib/python3.10/site-packages/sklearn/utils/validation.py", line 409, in check_consistent_length
    raise ValueError(
ValueError: Found input variables with inconsistent numbers of samples: [1753, 30]